test/rateLimiter.test.js

```diff
@@ -51,6 +53,61 @@ describe('shouldSend', function() {
     done();
   });
 
+  var setupDedupeTest = function() {
+    var client = new (TestClientGen())();
+    var now = (new Date()).getTime();    
+    var options = { 
+      startTime: now,
+      duplicateRateLimit: 5,
+    };
+    var rollbar = new Rollbar(options, client);
+    var rateLimiter = new RateLimiter(options);
+
+    return { rollbar: rollbar, rateLimiter: rateLimiter, client: client };
+  };
+
+  var createItem = function(setup, message, exception) {
+    setup.rollbar.handleUncaughtException(message, '', 0, 0, exception, null);
+
+    return setup.client.logCalls[setup.client.logCalls.length - 1].item;
+  }
+
+  it.only('asdfasdf', function (done) {
```

Comment:
I won't comment too much on this test as it appears to be a work in progress, but I would get rid of the magic numbers like `123123` and make that a constant like `now`, then the other ones can be offsets from there like `now + duplicateRateLimit - 1` and `now + duplicateRateLimit + 1`. `shouldSend` takes that second now parameter to make tests like this easy to write. But you lose the ability to use that by using the real current time in `setupDedupeTest` without getting a reference to that `now` value back to use in the tests.